.travis.yml

```diff
@@ -8,8 +8,11 @@ cache:
   directories:
   - node_modules
   - cache
+  - build/libdeps
 node_js:
   - 0.10
+  - 4
```

Comment:
I'm not sure about this.
We should probably concentrate supporting a single node version (6 or 4) and remove support for the others